README.md

```diff
@@ -125,7 +139,25 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S0        |    the    |     S2
+    S1        |   lazy    |     S2
+    S1        |   lazy    |     S1
+    S1        |  smelly   |     S2
+    S1        |  smelly   |     S1
+    S2        |    dog    |     S3
+    S2        |    cat    |     S3
+    S3        |    ate    |     S4
+    S3        |    ate    |     S5
+    S3        |    ran    |     S4
+    S3        |    ran    |     S5
+    S4        |  slowly   |     S5
+    S4        |  noisily  |     S5
+    S5        |    EOI    |     DONE
```

Comment:
So, imagine you are a state machine interpreter. It's a good steady job; predictable work.

Then someone gives you this table, and asks you to use it to analyze "the dog ran".

Easy, you say. I'm in my start state, S0, and my next word is "the". I just have to find that transition. Wait! There are two. Which do I take? I don't know! I'm deterministic! This isn't fair! 

If this were the original Star Trek, you'd then explode.

The thing I don't understand is that this doesn't correspond at all to your diagram...

# -4